### ISSUE TITLE
Add ability to change pylint's python interpreter when changing working python interpreter

### ISSUE THREAD

**Issue by chaNcharge** [Microsoft/vscode#40005]
- VSCode Version: 1.18.1
- OS Version: macOS 10.13.2

**Feature request for python extension:**

I frequently work with two different python versions, python 2.7.14 and 3.6.3. I have pylint installed on both versions; however, when I switch interpreters from 2 to 3 (and vice versa), pylint is stuck on the python 2 interpreter, which was the version pylint was most recently installed. I know it's possible because I made a shell script in my system's path that can change pylint python interpreters once, but it'd be even better if it it works within VSCode. 

Basically, what I want is when I change python interpreters, pylint will also change to the respective python interpreter.
